docs/kernel-policy.md

```diff
@@ -1,21 +1,37 @@
 # Firecracker's Kernel Support Policy
 
-Once officially supported kernel versions are supported for a **minimum of 2
-years**.
-
-We are validating the currently supported Firecracker releases as per
-[Firecracker’s release policy](../docs/RELEASE_POLICY.md). Starting with release
-`v1.0` each major and minor release will specify the supported kernel versions.
-Adding support for a new kernel version will result in a Firecracker release
-only if compatibility changes are required.
+Firecracker is tightly coupled with the guest and host kernels on which it is
+run. This document presents our kernel support policy which aims to help our
+customers choose host and guest OS configuration, and predict future kernel
+related changes.
+
+We are continuously validating the currently supported Firecracker releases (as
+per [Firecracker’s release policy](../docs/RELEASE_POLICY.md)) using a
+combination of:
+- host linux kernel versions 4.14, 5.10, and 6.1;
+- guest linux kernel versions 4.14, and 5.10.
+While other versions and other kernel configs might work, they are not periodically
+validated in our test suite, and using them might result in unexpected behaviour.
+Starting with release `v1.0` each major and minor release will specify the supported
+kernel versions.
```

Comment:
I think this is still missing. We have a description of our supported kernels, but this is not specific to each major and minor release.